src/main/java/com/cvsgo/service/UserService.java

```diff
@@ -85,4 +95,52 @@ public boolean isDuplicatedEmail(String email) {
     public boolean isDuplicatedNickname(String nickname) {
         return userRepository.findByNickname(nickname).isPresent();
     }
+
+    /**
+     * 회원 팔로우를 생성한다.
+     *
+     * @param user   로그인한 사용자
+     * @param userId 팔로잉할 사용자 ID
+     * @throws NotFoundUserException         해당하는 아이디를 가진 사용자가 없는 경우
+     * @throws BadRequestUserFollowException 본인을 팔로우하는 경우
+     * @throws DuplicateUserFollowException  이미 해당하는 회원 팔로우가 존재하는 경우
+     */
+    @Transactional(rollbackFor = Exception.class)
```

Comment:
해당 메소드에서는 throw하는 CheckedException이 없기 때문에 rollbackFor = Exception이 있으나 없으나 같아서 불필요할 것 같습니다!
```suggestion
    @Transactional
```